protocols/v2/noise-sv2/src/lib.rs

```diff
@@ -342,7 +342,11 @@ impl handshake::Step for Responder {
                     from_bytes(&mut in_msg);
                 match negotiation_message {
                     Ok(negotiation_message) => {
+                        println!("CCCCCCC");
```

Comment:
Were these just for testing, or do we want these in for production?

Reply:
Yes the are for testing and shouldn't be there, I don't know what happened